kong/db/strategies/postgres/init.lua

```diff
@@ -1248,6 +1254,22 @@ function _M.new(connector, schema, errors)
     }
   })
 
+  add_statement_for_export("select", {
```

Comment:
Could we add a comment to explain why?

Reply:
Compared with `add_statement_for_export("page_first", ...)`, this statement is not special. So I don't understand why we need a comment for this but not for "page_dirst". Additionally, `add_statment_for_export` is self-explanatory.

Reply:
Added.